src/main/java/org/springframework/samples/petclinic/PetClinicApplication.java

```diff
@@ -30,6 +30,7 @@
 
```

Comment:
The file has increased in 1 lines.
<!-- lookout footnote separator -->
_Comment made by the analyzer Dummy._